docs/lvmd.md

```diff
@@ -36,13 +36,14 @@ device-classes:
 
 The device-class settings can be specified in the following fields:
 
-| Name           | Type   | Default | Description                                                    |
-| -------------- | ------ | ------- | -------------------------------------------------------------- |
-| `name`         | string | -       | The name of a device-class.                                    |
-| `volume-group` | string | -       | The group where this device-class creates the logical volumes. |
-| `spare-gb`     | uint64 | `10`    | Storage capacity in GiB to be spared.                          |
-| `default`      | bool   | `false` | A flag to indicate that this device-class is used by default.  |
-
+| Name           | Type   | Default | Description                                                                        |
+| -------------- | ------ | ------- | ---------------------------------------------------------------------------------- |
+| `name`         | string | -       | The name of a device-class.                                                        |
+| `volume-group` | string | -       | The group where this device-class creates the logical volumes.                     |
+| `spare-gb`     | uint64 | `10`    | Storage capacity in GiB to be spared.                                              |
+| `default`      | bool   | `false` | A flag to indicate that this device-class is used by default.                      |
+| `stripe`       | uint   | -       | The number of stripes in the logical volume.                                       |
+| `stripe-size`  | uint   | -       | The amount of data that is written to one device before moving to the next device. |
```

Comment:
this description is from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/lv

Reply:
In json definition, `stripe-size` is string.
Would you fix this?

Reply:
Sorry, I'll fix it.